QA.md

```
@@ -120,21 +121,52 @@ last_verified_at: 2026-08-21
 
 #### 動作確認
 <details>
 <summary>動作確認エビデンス</summary>
 
-### **iOS で右エッジスワイプでも戻れる**: システムのナビゲーションバーを隠して独自ヘッダを使う画面で、画面左端からのスワイプでも前の画面に戻れる
+### **iOS で左エッジスワイプでも戻れる**: push 遷移した画面で、画面左端からのスワイプで前の画面に戻れる
+
+<details><summary>動作確認スクショ</summary>
+
+**確認日: 2026-08-25** (simtunnel リモート iOS Simulator / iOS 26。エディタからスワイプで戻った直後のホーム)
+<img src="https://pub-7f3469dd3e2e445b9b8ec2d1381b5ea8.r2.dev/bannzai/nikki/20260825/d619de59-47dd-4fa1-94ff-52f8d327b20e.jpg" width="320">
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)